synapse/lib/storm.py

```diff
@@ -248,6 +248,15 @@ def __init__(self, **opts):
 
         self.setOperFunc('addxref', self._stormOperAddXref)
 
+        # Cache compiled regex objects.
+        self._rt_regexcache = s_cache.FixedCache(1024, re.compile)
+        # Glob helpers
+        self._rt_glob_smark = '*'
```

Comment:
Since these dont have the need to be over-ridden by a subclass or smashed by an instance, the code is clearer with them directly in line especially since they're simple and most of the declared versions here are only used in one place later anyway ).  Swap these constants into their position below and we'll mainline!